### PR TITLE
chore(apps): add apps to marketplace

### PR DESCRIPTION
**Pull request by andrewg-xyz**
[screen-capture.webm](https://github.com/user-attachments/assets/72916707-54c8-493d-bca4-5dfd2a4fc9d6)

### CODE REVIEW

File: ui/static/images/icons/strimzi.png
Comment (jonnyborbs) [inline ui/static/images/icons/strimzi.png:1]: meta-commentary - have we established a standard for size/shape of logos we use in the marketplace? we really probably should - and i propose they should be square, nearly every software package will provide some form of a square logo as part of their brand/press kit. Worth checking with @Madeline-UX on this but I'd like to see it standardized

File: apps/zookeeper.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  repository: ghcr.io/defenseunicorns/packages/uds/zookeeper
+  tagline: |
+    **Reliable Distributed Coordination Made Simple**
+  description: |
+    Apache ZooKeeper is an open-source server that provides essential services for distributed systems. It offers centralized management of configuration information, naming services, distributed synchronization, and group services, which are crucial for maintaining consistency and coordination across distributed applications.
+
+    ZooKeeper significantly reduces the complexity and potential for errors in distributed systems by offering a robust, centralized solution for common coordination tasks. This allows developers to focus on core application logic rather than reimplementing complex distributed algorithms, resulting in more reliable and manageable distributed applications.
+
+    While ZooKeeper itself doesn't have a graphical user interface, it provides a set of primitives that applications can use to implement higher-level services. These include a hierarchical namespace for storing data (similar to a file system), watches for change notifications, and ephemeral nodes for implementing group membership protocols.
```
Comment (jonnyborbs) [inline apps/zookeeper.yaml:15]: Feel like we could drop all of line 15
```suggestion
```

**Review by jonnyborbs — COMMENTED** [review]
I fact checked the 5 new app descriptions and reviewed them for glaring errors and unnecessary information. What I didn't do is ensure they align on tone and voice, because I'm not sure if that's something we intend to do for every app long term

I also have questions about aspect ratios for logos which I left on the strimzi image and also dropped in slack

**Comment by marshall007** [comment]
I approved but I think we'll want @Madeline-UX to rework the icons eventually.

**Pull request merged**